src/include/netloc_util.h

```diff
@@ -40,4 +42,9 @@ int MPIR_Netloc_get_switches_at_level(netloc_topology_t netloc_topology,
                                       MPIR_Netloc_network_attributes attributes, int level,
                                       netloc_node_t *** switches_at_level, int *switch_count);
 
+int MPIR_Netloc_get_hostnode_index_in_tree(MPIR_Netloc_network_attributes attributes,
```

Comment:
This must belong in the next commit, right? This function doesn't exist yet.

Reply:
Oh sorry, i seem to have mixed this commit with the next one, will move it to the next commit.

Reply:
Fixed this, along with a couple of typos in commit messages.